### ISSUE TITLE
Question: What are some heuristic rules for initialization?

### ISSUE THREAD

**Issue by svilupp** [biaslab/ReactiveMP.jl#145]
It would be awesome to provide some simple tips for `initmessages` and `initmarginals` keywords in the `inference()`.

It was briefly covered [HERE](https://discourse.julialang.org/t/simple-linear-regression-with-reactivemp-rulemethoderror/81526/6), but it would be great to formalize the guidance a bit for people unfamiliar with the implementation.

**Comment by albertpod** [comment]
Should we transfer this issue to `RxInfer.jl`?

**Comment by bvdmitri** [comment]
Yes, the `inference` function is going to be transferred to the `RxInfer.jl`.
